node/src/blockchain/blockchain_interface.rs

```diff
@@ -383,10 +416,57 @@ where
 
 impl<T> BlockchainInterfaceNonClandestine<T>
 where
-    T: Transport + Debug + 'static,
+    T: 'static + BatchTransport + Debug,
+{
+    fn find_largest_transaction_block_number(
+        &self,
+        response_block_number: u64,
+        transactions: &Vec<BlockchainTransaction>,
+    ) -> u64 {
+        if transactions.is_empty() {
+            response_block_number
+        } else {
+            transactions
```

Comment:
Is there already a maximum function somewhere in the Rust standard library (or perhaps in `itertools`) that could make this simpler?

Reply:
Hopefully the change is more readable